lib/session.js

```diff
@@ -447,10 +447,14 @@ function hash(sess) {
  */
 function compatMaxage(opts) {
   if (opts) {
-    opts.maxage = opts.maxage === undefined
-      ? opts.maxAge
-      : opts.maxage;
-    delete opts.maxAge;
+    if (require('koa')) {
```

Comment:
why should we do this?

how about just set `opts.maxAge` to keep forward compatbility

```js
opts.maxAge = opts.maxage === undefined
  ? opts.maxAge
  : opts.maxage;
delete opts.maxage;
```